README.md

```diff
@@ -23,6 +23,14 @@ Run this command to install the component:
   yarn add @kyma-project/documentation-component
   ```
 
+## Quick start
```

Comment:
```suggestion
### Quick start
```

Reply:
why? it is separate section and not a subsection of installation

Reply:
I added it to be compliant with the template we use. Perhaps you can add it as a subsection to a different section if it fits better somewhere else.